switchbot_ros/src/switchbot_ros/switchbot.py

```diff
@@ -4,15 +4,26 @@
 import os.path
 import requests
 
+import os
+import time
+import hashlib
+import hmac
+import base64
+import uuid
+
 
 class SwitchBotAPIClient(object):
     """
     For Using SwitchBot via official API.
     Please see https://github.com/OpenWonderLabs/SwitchBotAPI for details.
     """
-    def __init__(self, token):
-        self._host_domain = "https://api.switch-bot.com/v1.0/"
+    def __init__(self, token, secret="secret_none"):
+        if secret=="secret_none":
```

Comment:
As I commented in launch file, this line would be
```python
secret=""
```
and 
```python
if not secret:
```

And I request to add printing the version information like 
```python
rospy.loginfo("Using SwitchBot API v1.0")
```

Reply:
switchbot.py was modified to match the empty `secret` with adding an instance `self.api_version`.
And I change the instance name `host_domain` back to the original name `_host_domain`.

``` python
    def __init__(self, token, secret=""):
        if not secret:
          self.api_version = "v1.0"
        else:
          self.api_version = "v1.1"
        self._host_domain = "https://api.switch-bot.com/" + self.api_version + "/"

```

https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/491/commits/dc12bac96f632ccd1cc78f94ab69c8ae24475cf2#diff-396497c868baf8630a5e57ccf98059c7d6f1d45e91faccfe0917e1012f124191R20-R25

---

The method `print_baseurl()` was changed to `print_apiversion()` and fixed the unmatched name of the internal variables.

``` python
    def print_apiversion(self):
        if self.bots is None:
            return
        
        apiversion_str = 'Using SwitchBot API ';
        apiversion_str += self.bots.api_version;
        rospy.loginfo(apiversion_str)        
```

https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/491/commits/dc12bac96f632ccd1cc78f94ab69c8ae24475cf2#diff-1690cbebb7bf92ad5c7be4524a54f4c0566916a4da9a6b0c493666e891db80c9R72-R78

https://github.com/jsk-ros-pkg/jsk_3rdparty/pull/491/commits/0b1a8029fc6db9ee6ccb0a25eb48b5f763fc5a6a#diff-1690cbebb7bf92ad5c7be4524a54f4c0566916a4da9a6b0c493666e891db80c9R76-R78